src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/PlaceholderTest.kt

```diff
@@ -98,11 +96,6 @@ class PlaceholderTest {
                     }
                     assertEquals("Only one item is selected", selectedItems.size, 1)
                     assertTrue("Selected module is correct", selectedItems[0].contains("Viinaharava", true))
-                    keyboard {
-                        escape()
-                        enterText("'") // a character that won't match anything
-                    }
-                    assertEquals("Nothing should be selected", selectedItems.size, 0)
```

Comment:
In new version of IntelliJ, the selected item stays selected, so this check is no longer needed.